firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/snapshot.ts

```diff
@@ -60,30 +60,22 @@ export function buildLatestSnapshotViewQuery(
     --   event_id: The id of the event that triggered the cloud function mirrored the event.
     --   data: A raw JSON payload of the current state of the document.
     --   document_id: The document id as defined in the Firestore database
-    SELECT
```

Comment:
For early adoption, would it be worth considering a configuration option to allow users to opt-in to the new Query. This could then be removed at a later date if the new format is widely adopted/successful?